Monika After Story/game/script-topics.rpy

```diff
@@ -4703,6 +4703,71 @@ label monika_write:
     m "Thanks for listening~"
     return
     
+init 5 python:
+    for key in ['Psychological Quiz']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('Monika_Psychological_Quiz')
+    monika_random_topics.append('Monika_Psychological_Quiz')
+
+label Monika_Psychological_Quiz:
```

Comment:
While this works syntactically, the convention we use for labels is all lowercase. I'm **not** going to mandate a change personally (cause i'll just do it myself), but if another reviewer requests this, then you'll have to change it.